TaskModules/powershell/TaskModuleSqlUtility/SqlPackageOnTargetMachines.ps1

```diff
@@ -468,7 +468,42 @@ function Execute-DacpacDeployment
     Write-Verbose -Verbose $sqlPackageArguments
     
     Write-Verbose "Executing command: $sqlPackage $sqlPackageArguments"
-    Execute-Command -FileName "$sqlPackage"  -Arguments $sqlPackageArguments
+    if($PSVersionTable.PSVersion.Major -lt 4)
+    {
+        Run-Command "`"$sqlPackage`" $sqlPackageArguments"
+    }
+    else
+    {
+        Execute-Command -FileName "$sqlPackage"  -Arguments $sqlPackageArguments
```

Comment:
As Run-Command itself has a modular logic to run command on cmd, i think i should not change that instead should introduce a new method which depends on condition of powershell version call run-command or execute-command.